hawc/apps/assessment/templates/assessment/_assessmentvalue_list.html

```diff
@@ -0,0 +1,33 @@
+{% if object_list.count > 0 %}
+<h3>Assessment Values</h3>
+<table class="table table-sm table-striped">
+    <colgroup>
+        <col style="width: 30%;">
+        <col style="width: 50%;">
+        <col style="width: 20%;">
+    </colgroup>
+    <thead>
+        <tr>
+            <th>Value</th>
+            <th>Comments</th>
+            <th>Actions</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for object in object_list %}
+        <tr>
+            <td>{{object.value}}</td>
+            <td>{{object.comments}}</td>
+            <td>
```

Comment:
we need to selectively show/hide that entire column depending on permission - if you can't edit, we shouldn't show the thead either